Userland/Applets/Audio/main.cpp

```diff
@@ -34,18 +35,18 @@ class AudioWidget final : public GUI::Widget {
 public:
     static ErrorOr<NonnullRefPtr<AudioWidget>> try_create()
     {
-        Vector<VolumeBitmapPair, 5> volume_level_bitmaps = {
+        auto volume_level_bitmaps = TRY(FixedArray<VolumeBitmapPair>::try_create{
             { 66, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/audio-volume-high.png")) },
             { 33, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/audio-volume-medium.png")) },
             { 1, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/audio-volume-low.png")) },
             { 0, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/audio-volume-zero.png")) },
             { 0, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/audio-volume-muted.png")) }
-        };
+        }));
         return adopt_nonnull_ref_or_enomem(new (nothrow) AudioWidget(move(volume_level_bitmaps)));
     }
 
 private:
-    AudioWidget(Vector<VolumeBitmapPair, 5> volume_level_bitmaps)
+    AudioWidget(FixedArray<VolumeBitmapPair> volume_level_bitmaps)
```

Comment:
Accompanying the change above:
```suggestion
    AudioWidget(Array<VolumeBitmapPair, 5> volume_level_bitmaps)
```

Reply:
You're right, that's a good idea, it will make us not allocate at all.